Packs/FeedAzure/Integrations/FeedAzure/FeedAzure.py

```diff
@@ -0,0 +1,325 @@
+                results.append(
+                    self.build_ip_indicator(address,
+                                            azure_name=indicator_metadata['name'],
+                                            azure_id=indicator_metadata['id'],
+                                            azure_region=indicator_metadata['region'],
+                                            azure_platform=indicator_metadata['platform'],
+                                            azure_system_service=indicator_metadata['system_service'])
+                )
+
+        return results
+
+    def build_iterator(self) -> List:
+        """Retrieves all entries from the feed.
+        Returns:
+            A list of objects, containing the indicators.
+        """
+        try:
+            download_link = self.get_azure_download_link()
+            values_from_file = self.get_download_file_content_values(download_link)
+            results = self.extract_indicators_from_values_dict(values_from_file)
+
+            return results
+
+        except (requests.exceptions.SSLError, requests.ConnectionError, requests.exceptions.HTTPError) as err:
+            demisto.debug(str(err))
+            raise Exception(ERROR_TYPE_TO_MESSAGE[err.__class__] + str(err))
+
+        except RuntimeError as err:
+            demisto.debug(str(err))
+            raise RuntimeError(F'Could not fetch download link from Azure')
+
+        except ValueError as err:
+            demisto.debug(str(err))
+            raise ValueError(f'Could not parse returned data to Json. \n\nError massage: {err}')
+
+
+def test_module(client: Client) -> Tuple[str, Dict, Dict]:
+    """Test the ability to fetch Azure file.
+    Args:
+        client: Client object.
+    Returns:
+        str. ok for success, relevant error string otherwise.
+    """
+    try:
+        download_link = client.get_azure_download_link()
+        client.get_download_file_content_values(download_link)
+
+    except (requests.exceptions.SSLError, requests.ConnectionError, requests.exceptions.HTTPError) as err:
+        demisto.debug(str(err))
+        raise Exception(ERROR_TYPE_TO_MESSAGE[err.__class__] + str(err))
+
+    return 'ok', {}, {}
+
+
+def get_indicators_command(client: Client) -> Tuple[str, Dict, Dict]:
+    """Retrieves indicators from the feed to the war-room.
+
+    Args:
+        client (Client): Client object configured according to instance arguments.
+
+    Returns:
+        Tuple of:
+            str. Information to be printed to war room.
+            Dict. The raw data of the indicators.
+    """
+    limit = int(demisto.args().get('limit')) if 'limit' in demisto.args() else 10
+
+    indicators, raw_response = fetch_indicators_command(client, limit)
+
+    human_readable = tableToMarkdown('Indicators from Azure Feed:', indicators,
+                                     headers=['value', 'type'], removeNull=True)
+
+    return human_readable, {}, {'raw_response': raw_response}
```

Comment:
Is there any particular reason you are nesting the raw response into another dictionary here? It seems superfluous. If `fetch_indicators_command` doesn't return a dictionary and you want it to be a dictionary it seems like you should nest it in a dictionary there before returning it but it's up to you.